packages/studio-ui/src/components/PropEditor.tsx

```diff
@@ -48,9 +49,18 @@ export default function PropEditor({
     <div className="flex mb-2 text-sm grow flex-col">
       <div className="flex relative">
         {isNested && renderBranchUI()}
-        <label className="pb-1" id={labelTooltipId} htmlFor={inputId}>
-          {propName}
-        </label>
+        <div className="flex flex-row">
+          <label className="pb-1" htmlFor={inputId}>
+            {propName}
+          </label>
+          {tooltip && (
+            <Info
+              id={labelTooltipId}
```

Comment:
nit: can the variable name and value be updated now that the ID is no longer for a label element?